Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -85,6 +85,55 @@ Validate that the Cilium Local Redirect Policy CRD has been registered.
 	   [...]
 	   ciliumlocalredirectpolicies.cilium.io              2020-08-24T05:31:47Z
 
+.. note::
+
+    Local Redirect Policy supports either the socket-level loadbalancer or the tc loadbalancer at the veth interface.
```

Comment:
Lets not mentioned veth interface since it can be confusing for folks with netkit. Also, the description is mostly implementation details (targeted at developers but not end userd). I would love if we instead could focus in which scenario/use-case a user should apply which config?